Fix create group

Neriderc opened the pull request:
Previous changes to group of secret key have meant key is not being asked for during creation of group. This means new group is created but user is not added.

This small change fixes #87

Neriderc commented:
Editing a group name also doesn't work. There might need to be more changes to fully separate the invite key away from the group secret key.

nrobinaubertin commented:
I changed the target branch to be `next` instead of master and fixed the conflicts

Neriderc commented:
> I changed the target branch to be `next` instead of master and fixed the conflicts

Ah crap it has been too long, I'd forgotten I was supposed to target `next`, sorry!
